src/resources/help/en_US/relnotes3.0.2.html

```diff
@@ -33,7 +33,9 @@
   <li>Corrected a bug with the Uintah reader where it would not load because the libxml2 could not be found.</li>
   <li>Corrected a bug where the GUI plot list goes blank on macOS.</li>
   <li>Corrected a bug where the VTK reader incorrectly set topological dimension of a dataset to 0, making the dataset undrawable by VisIt. This occured in a multiblock case where the first block contained neither points nor cells.</li>
-  <li>Corrected a bug where the OriginalZoneLabels and OriginalNodeLabels variables appeared twice in the menu for Mili files.</li>
+  <li>Corrected a bug with highlighting zones picked by their global ids. The highlighted cells were incorrect or non-existent before.</li>
```

Comment:
The merge of the 3.0.2 release notes isn't quite correct. I made a few changes in 3904 and 3905 that you didn't pick up.

Reply:
I think this is resolved here now too.